### PR TITLE
hwloc: add v2.11.1

### PR DESCRIPTION
**Pull request by drkrynstrng**
Adding latest releases v2.11.1 and v2.11.0.

### CODE REVIEW

**Comment by bgoglin** [comment]
I would rather skip 2.11.0 since 2.11.1 is the same without the broken bash completion.

**Comment by drkrynstrng** [comment]
Okay, I removed v2.11.0.

File: var/spack/repos/builtin/packages/hwloc/package.py [hunk 1]
```diff
@@ -106,6 +100,8 @@ class Hwloc(AutotoolsPackage, CudaPackage, ROCmPackage):
         "oneapi-level-zero", default=False, description="Support Intel OneAPI Level Zero devices"
     )
 
+    depends_on("c", type="build")
+    depends_on("cxx", type="build")
```
Comment (bgoglin) [inline var/spack/repos/builtin/packages/hwloc/package.py:104]: cxx isn't strictly required, it's only used during examples build, but configure disables it if no C++ compiler is found.

**Review by bgoglin — APPROVED** [review]
Otherwise OK.

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**